recipes/sentry-breakpad/all/conanfile.py

```diff
@@ -0,0 +1,143 @@
+import os
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.28.0"
+
+
+class SentryBreakpadConan(ConanFile):
+    name = "sentry-breakpad"
+    description = "A set of client and server components which implement a crash-reporting system."
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/getsentry/breakpad"
+    license = "Apache-2.0"
+    topics = ("conan", "breakpad", "error-reporting", "crash-reporting")
+    provides = "breakpad"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+    }
+    exports_sources = "CMakeLists.txt", "patches/*"
+    generators = "cmake"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("linux-syscall-support/cci.20200813")
+
+    def validate(self):
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 11)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.configure()
+        return self._cmake
+
+    def _patch_sources(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+
+        # FIXME: convert to patches
+        import textwrap
+
+        files_to_patch = [
```

Comment:
You can take the patch from my PR, it should work.
https://github.com/conan-io/conan-center-index/pull/5639/files#diff-46f93bc4248cb2a5f042278e731395de3ffdbdc16f1c3e78935b713a652d28f7